tests/integration/test_store.py

```diff
@@ -607,7 +607,7 @@ def send_buckets(buckets):
 
 
 @pytest.mark.parametrize("violating_bucket", [2.0, 3.0])
-def test_rate_limit_metrics_buckets(
+def test_enforce_transaction_rate_limit_on_metric_buckets(
```

Comment:
Unrelated: I renamed these tests while looking into some other flakiness, and found it hard to distinguish between them.